test/engines/pyrgg_test.py

```diff
@@ -561,29 +539,17 @@
 76
 >>> testfile_3['graph']['edges'][1]['weight']
 15
->>> testfile_3['properties']['signed']
-False
 >>> testfile_3['properties']['directed']
 True
->>> testfile_3['properties']['self_loop']
-True
->>> testfile_3['properties']['multigraph']
```

Comment:
Please revert **multigraph** tests

Reply:
fixed in [25f71d4](https://github.com/sepandhaghighi/pyrgg/pull/173/commits/25f71d4b3f5fb271376342350900f6a59fa5fc50).